src/blockdata/script.rs

```diff
@@ -43,10 +43,16 @@ use util::key::PublicKey;
 /// A Bitcoin script
 pub struct Script(Box<[u8]>);
```

Comment:
Why not to use this opportunity and derive standard things for Script? We can do `#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Default)]`

Reply:
I mean we totally could do that here, but it's up to @stevenroose if he wants it in, it's kinda unrelated.

Reply:
It probably belongs in another PR. Though I definitely like those proposed changes.